src/aot/aot_main.cpp

```diff
@@ -106,18 +111,34 @@ int main(int argc, char* argv[])
     return 1;
 
   BPFtrace bpftrace(std::move(output));
-  int err = aot::load(bpftrace, argv[optind]);
+  int err = aot::load(bpftrace, argv[0]);
   if (err) {
     LOG(ERROR) << "Failed to load AOT script";
     return err;
   }
 
+  // Signal handler that lets us know an exit signal was received.
```

Comment:
Why do we need all these signal handlers?

Reply:
I'm actually not too sure - I copied this code from the end of main() in main.cpp. I figured aot will need to duplicate most of the behavior surrounding these signals since we want it to behave basically the same as regular bpftrace. But I can remove them if they seem unnecessary.

Reply:
Sure - created run_bpftrace.h/cpp to handle this.